src/pages/Create/Note.tsx

```diff
@@ -1,23 +1,43 @@
 import {
   useDeskproAppClient,
   useDeskproAppEvents,
+  useDeskproLatestAppContext,
   useInitialisedDeskproAppClient,
 } from "@deskpro/app-sdk";
-import { useState } from "react";
+import { useState, useCallback } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import { InputWithTitle } from "../../components/InputWithTitle/InputWithTitle";
 import { Button, P8, Stack } from "@deskpro/deskpro-ui";
 import { createNote } from "../../api/api";
 
 export const CreateNote = () => {
   const { client } = useDeskproAppClient();
+  const { context } = useDeskproLatestAppContext();
   const navigate = useNavigate();
   const { taskId } = useParams();
 
   const [submitting, setSubmitting] = useState<boolean>(false);
   const [note, setNote] = useState<string>("");
   const [error, setError] = useState<string | null>(null);
 
+  const onCreate = useCallback(() => {
+    if (!client || !context?.settings || !taskId) {
+      return;
+    }
+
+    setSubmitting(true);
+    setError(null);
+
+    if (note.length === 0) {
+      setError("Note cannot be empty");
```

Comment:
should `setSubmitting(false)` be called before returning otherwise I think this gets the UI stuck/locked.

Reply:
fixed